static/styles/components.scss

```diff
@@ -101,3 +101,27 @@ a.no-underline:hover {
 .flex-row .field + .field {
     margin-left: 10px;
 }
+
+/* -- draggable "grip" icon --*/
+.grip-icon {
+    content: '....';
+    width: 10px;
+    height: 20px;
+    display: inline-block;
+    overflow: hidden;
+    line-height: 5px;
+    padding: 3px 4px;
+    cursor: move;
+    vertical-align: middle;
+    margin-top: -.7em;
+    margin-right: .3em;
+    font-size: 12px;
+    font-family: sans-serif;
+    letter-spacing: 2px;
+    color: #cccccc;
+    text-shadow: 1px 0 1px black;
+}
+
+.grip-icon::after {
+    content: '.. .. .. ..';
```

Comment:
Have you checked [Font awesome](https://fontawesome.com/v4.7.0) to check if there was already an icon that could be suited for grip usage? [Vertical ellipsis](https://fontawesome.com/v4.7.0/icon/ellipsis-v) and [bars](https://fontawesome.com/v4.7.0/icon/bars) seem appropriate for me (probably worth starting a discussion on chat.zulip.org though)

Reply:
Yeah, we're using an older version of font awesome that doesn't support those icons. Figured it was easier to create something than to upgrade font awesome for now.

Reply:
We're using font awesome v4.7.0. Those icons are from v4.7.0.

Reply:
Whoops—I couldn't get them to work and assumed we were on an older version, clearly I was wrong. I'll update this to use font awesome. Thanks for catching!